internal/repository/integration/mongo/employee_test.go

```diff
@@ -0,0 +1,376 @@
+//go:build integration
+// +build integration
+
+package mongo
```

Comment:
this test is not working locally "out of the box" and there are no instructions in readme on proper set up.

```
2024/07/03 16:41:59 Error while getting config: JWT_TOKEN_SECRET is empty
REST_PORT is empty
GRPC_PORT is empty
ADDRESS is empty
DB_HOST is empty
DB_PORT is empty
DB_USER is empty
DB_PASSWORD is empty
DB_USER is empty
DB_TIMEOUT is empty
DB_TIMEOUT must be an integer
DB_MAX_CONNECTIONS is empty
DB_MAX_CONNECTIONS must be an integer
```